components/MusicControl/index.js

```diff
@@ -0,0 +1,28 @@
+import { StyledVolumeControl } from "../../styles/StyledVolumeControl";
+import { StyledMusicButton } from "../../styles/StyledMusicButton";
+
+export default function MusicControl({
+  music,
+  handleIncreaseMusicVolume,
+  handleDecreaseMusicVolume,
+}) {
+  return (
+    <>
+      <StyledVolumeControl>
+        <StyledMusicButton
+          onClick={handleIncreaseMusicVolume}
+          disabled={music.volume >= 1}
+        >
+          Music +
```

Comment:
Have you tested this text with a screen reader? To be sure, I would add an `aria-label` to both `StyledMusicButton`s.